### PR TITLE
Branch name bswhite selects beat-aml

### PR DESCRIPTION
**Pull request by bswhite**

### CODE REVIEW

File: config/prod/bwhite-beat-aml-bioc-again.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Provision an EC2 instance
+	template_path: remote/managed-ec2-linux-v1.j2
```
Comment (thomasyu888) [inline config/prod/bwhite-beat-aml-bioc-again.yaml]: I think you have a bunch of tabs.

File: config/prod/bwhite-beat-aml-bioc-new.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2-ubuntu-v3.j2
```
Comment (thomasyu888) [inline config/prod/bwhite-beat-aml-bioc-new.yaml]: Remove this file

**Comment by bswhite** [comment]
@thomasyu888 you're looking at the file bwhite-beat-aml-bioc-again.yaml, right?

It has 
template_path: remote/managed-ec2-linux-v1.j2
which is the same as the template you sent me
https://github.com/Sage-Bionetworks/sandbox-provisioner/pull/3/files

I don't understand the problem.

File: config/prod/bwhite-beat-aml-bioc-new.yaml (hunk 2 above)
Comment (bswhite) [reply config/prod/bwhite-beat-aml-bioc-new.yaml]: The request is in bwhite-beat-aml-bioc-again.yaml (which uses template_path: remote/managed-ec2-linux-v1.j2 and should be correct) and not bwhite-beat-aml-bioc-new.yaml (which uses the incorrect template_path: remote/managed-ec2-ubuntu-v3.j2). Is bwhite-beat-aml-bioc-again.yaml ready to go? Do I need to remove bwhite-beat-aml-bioc-new.yaml?

Comment (thomasyu888) [reply config/prod/bwhite-beat-aml-bioc-new.yaml]: Yes you need to remove the bwhite-beat-aml...new.yaml.

**Comment by thomasyu888** [comment]
There is nothing wrong with your bwhite....again.yaml.  also you will have to merge in the sage master branch as there seem to have been changes to it.

File: config/prod/bwhite-beat-aml-bioc-again.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  Department: "CompOnc"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "beat-aml"
+  # The resource owner
+  OwnerEmail: "brian.white@sagebase.org"
+
+  # Default account settings, leave alone unless you plan to override
+  VpcName: "sandcastlevpc"
+  KeyName: "sandbox"
+  AMIId: "ami-08333c98cd3b5ada7"
```
Comment (thomasyu888) [inline config/prod/bwhite-beat-aml-bioc-again.yaml]: I think you have a trailing white space on this line. There can't be any trailing white spaces.

**Comment by bswhite** [comment]
Thanks for all your help, @thomasyu888 . But I still have not received an email from aws.sandbox with details on how to connect to the node. Am I missing some additional step?

**Comment by thomasyu888** [comment]
There is an issue with spinning up your node.... Not sure how to fix

**Comment by ahayden** [comment]
This is repeat of a stack that built successfully, right? The error I saw was that the stack name was not unique, so I deleted the stack through the CF console and rebuilt, but it failed again. The difference I see between the one that succeeded and the one that failed is the 'template_path' param... Let's try 
`# Provision an EC2 instance`
`template_path: remote/managed-ec2-ubuntu-v3.j2`
*instead* of 
`# Provision an EC2 instance`
`template_path: remote/managed-ec2-linux-v1.j2`
@bswhite can you recommit with this change if you agree?

**Comment by bswhite** [comment]
@ahayden , yes I can make that change but @thomasyu888 intentionally had me switch the template path to reflect the latest template PR here:
https://github.com/Sage-Bionetworks/sandbox-provisioner/pull/3/files

And, yes, it is a repeat of a stack that built successfully previously.

**Comment by ahayden** [comment]
Well... Ubuntu vs. AMZN Linux install dependencies differently, which I think is causing the build to fail, so let's try despite what @thomasyu888 says

**Comment by bswhite** [comment]
OK, done, @ahayden ... just pending a review.

**Comment by thomasyu888** [comment]
Ah! @bswhite. It was my oversight. I thought the AMI you specified was the default amazon Linux AMI. notice there is a distro field in the template, you will have to update that to Ubuntu for this new template to work.  The older version will work for now since there are still instances that are still running that used these older versions. Once all of these instances are terminated, that older template will be removed I think.

**Pull request merged**